### PR TITLE
Move account switch inside page, improved select dates

### PR DESCRIPTION
**Pull request by ptbrowne**
- Account switch is no longer in the sidebar and has been moved
inside the transactions and categorization page

- Date selector has been improved to facilitate comparing years and months

### CODE REVIEW

File: src/ducks/transactions/TransactionsPage.jsx [hunk 1]
```diff
@@ -275,16 +287,20 @@ class TransactionsPage extends Component {
             period={this.state.currentMonth}
             onChange={this.handleChangeMonth}
           />
-          <TransactionTableHead />
+          {filteredTransactions.length > 0 ? (
+            !isMobile ? (
```
Comment (kosssi) [inline src/ducks/transactions/TransactionsPage.jsx]: `!isMobile && <TransactionTableHead />` pourrait fonctionner ?

Comment (kosssi) [reply src/ducks/transactions/TransactionsPage.jsx]: pareil https://github.com/cozy/cozy-banks/pull/101/files#diff-ceb283ed4feb6c879d56b055bed20371R273 et https://github.com/cozy/cozy-banks/pull/101/files#diff-ceb283ed4feb6c879d56b055bed20371R277

Comment (ptbrowne) [reply src/ducks/transactions/TransactionsPage.jsx]: fixed in https://github.com/cozy/cozy-banks/pull/101/commits/d651abc3b74a1c2455330d74991275a8f368f1b3

File: src/components/Select/index.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <option key={option.value} value={option.value}>
+          {option.name}
+        </option>
+      ))}
+    </select>
+    <SmallArrow />
+  </span>
+)
+
+export default translate()(props => <Select {...props} />)
```
Comment (drazik) [inline src/components/Select/index.jsx]: Why not just `export default translate()(Select)` here ?
Also, it seems you are not using the `t` prop, so maybe the `translate` HOC is not required ?

File: src/components/Select/styles.styl [hunk 3]
```diff
@@ -0,0 +1,19 @@
+@require 'settings/palette'
+
+.Select
+    appearance none
```
Comment (drazik) [inline src/components/Select/styles.styl]: Did you test this on IE ? I remember it doesn't support `appearance`, so it renders the system arrow (see https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-appearance). There are workarounds for IE : https://stackoverflow.com/questions/1895476/how-to-style-a-select-dropdown-with-css-only-without-javascript/13968900#13968900

File: src/ducks/account/AccountSwitch.jsx [hunk 4]
```diff
@@ -276,8 +277,7 @@ class AccountSwitch extends Component {
       breakpoints: { isMobile, isTablet, isDesktop }
     } = this.props
     const { open } = this.state
-    let { accounts, groups, groupsDocs } = this.props
-    const isFetching = isLoading(accounts) || isLoading(groupsDocs)
+    let { accounts, groups /* groupsDocs */ } = this.props
```
Comment (drazik) [inline src/ducks/account/AccountSwitch.jsx]: There is an unneeded comment here I think

File: src/components/Select/index.jsx (hunk 2 above)
Comment (ptbrowne) [reply src/components/Select/index.jsx]: true, that's weird

File: src/components/Select/styles.styl (hunk 3 above)
Comment (ptbrowne) [reply src/components/Select/styles.styl]: OK thanks !

**Pull request merged**